.docs/content/armonik/getting-started.md

```diff
@@ -0,0 +1,44 @@
+# Getting Started with ArmoniK
+
+Welcome to the ArmoniK documentation! This guide will help you get started with installing, configuring, and deploying ArmoniK on various platforms. Below, you'll find links to detailed guides for each step.
+
+First you'll have to install ArmoniK's [Prerequisites](../getting-started/installation/prerequisites.md).
+
+Then you will have the choice to either deploy on your local machine, on an existing Kubernetes cluster, on AWS, or on GCP. The steps to do so are available in the following sections. The last step is to check that everything is working.
+
+## Installation on your Machine
+
+Learn how to set up ArmoniK on your machine:
+
+- [Local Deployment](../getting-started/installation/local.md): Deploy ArmoniK locally for development purposes.
+- [Kubernetes Deployment](../getting-started/installation/kubernetes.md): Install ArmoniK on a Kubernetes cluster.
```

Comment:
This link should be removed i guess

Reply:
WSL and K3s installation was split since our deployment work on any Kubernetes

Reply:
I removed the link to Kubernetes in the getting started as we point to it in the local deployment doc